### ISSUE TITLE
Web interface no longer loads.

### ISSUE THREAD

**Issue by dadrepus** [xteve-project/xTeVe#165]
#### Please first read the [documentation](https://github.com/xteve-project/xTeVe-Wiki/blob/master/en/configuration.md). Problems affecting the client, e.g. Plex or Emby should be reported there in the forum.
![Screen Shot 2020-08-07 at 11 51 50 AM](https://user-images.githubusercontent.com/22350089/89664103-a3a5a700-d8a4-11ea-88c2-a9b64ddbcdbe.png)

**Describe the bug**

**Server (please complete the following information):**
- OS [e.g. Linux - Ubuntu]
- Arch [e.g. Intel 64 Bit]
- xTeVe Version and Build [to be found in the xTeVe log]

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by MCTyler** [comment]
What are you reporting...Did you read the instructions? https://github.com/xteve-project/xTeVe-Wiki/blob/master/en/configuration.md

**Comment by dadrepus** [comment]
Yes, I read them. It is not like it didn’t work initially. Everything installed fine and worked for several days. Now the web interface does not load even after a reinstall from this site.

**Comment by dadrepus** [comment]
How do I permanently remove it?

**Comment by MCTyler** [comment]
You still haven't included any relevant information as listed in the "Make a Report" outline....How do I reproduce the problem like this?
![image](https://user-images.githubusercontent.com/6390898/89735696-ffe60380-da29-11ea-8381-a763eb407051.png)
